wasabi/__init__.py

```diff
@@ -1,12 +1,12 @@
 # coding: utf8
 from __future__ import unicode_literals
 
```

Comment:
There are also still some remaining `coding` lines in a couple places?
```suggestion
```

Reply:
Oh okay I missed that. Will update them in a few!